packages/ovm/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "all": "yarn clean && yarn build && yarn test && yarn fix && yarn lint",
     "test:govm": "waffle waffle-config.json && mocha --require source-map-support/register --require ts-node/register 'test/govm/*.spec.ts' --timeout 60000 --exit",
-    "test": "waffle waffle-config.json && mocha --require source-map-support/register --require ts-node/register 'test/contracts/*.spec.ts' --timeout 8000 --exit",
+    "test": "waffle waffle-config.json && mocha --require source-map-support/register --require ts-node/register 'test/contracts/*.spec.ts' 'test/app/*.spec.ts' --timeout 8000 --exit",
```

Comment:
Can we use `test/**/*.spec.ts` instead of writing both out?

Reply:
Unfortunately we can't match on everything in `test` because that would match `test/govm` which we want to run separately with `test:govm`. We might be able to remove `govm` tests in a separate PR if we decide to table `govm` for the time being.

I did at least [combine app and contracts into one](https://github.com/ethereum-optimism/optimism-monorepo/pull/21/commits/904311c97954492b0eab688524fbc30da12e066b) though 👍